frontstage/templates/errors/validation_error.html

```diff
@@ -0,0 +1,13 @@
+{% block main %}
```

Comment:
👍  for having this as a separate component
Would there we a way to make it more general so we can use it for the rest of our validation too and just pass in variables; `title`, `message` `focus_on` etc

Reply:
done